WordPress/UITests/Flows/LoginFlow.swift

```diff
@@ -32,4 +32,13 @@ class LoginFlow {
             .proceedWithValidPassword()
             .continueWithSelectedSite(title: selectedSiteTitle)
     }
+
+    // Login without selecting site
+    @discardableResult
+    static func loginWithoutSelectingSite(siteUrl: String, email: String, password: String) throws -> LoginEpilogueScreen {
```

Comment:
It looks like we can remove a couple of parameters here:
* We don't need to pass `siteUrl` since no site will be selected
* We also don't need to pass `password` as the function calls `.proceedWithValidPassword()` 

I also noticed we have other similar cases of unnecessary parameters within this file, maybe it's an opportunity to clean that up? WDYT? 😬

Reply:
thanks for catching that, such a rookie mistake (i was also convinced that i removed the `siteUrl` but i did not)! 🤦‍♀️ removed in 000cfc4c967c257de830f6868079a8608cac6d75

for the clean up of the `loginFlow` file, i think would be better to do that in a separate PR there are enough changes in this _simple_ PR already (updating those would mean updating all tests) 😬

Reply:
Yep, it makes sense!